Sources/Build/describe().swift

```diff
@@ -66,18 +74,26 @@ public func describe(prefix: String, _ conf: Configuration, _ modules: [Module],
 
             let node = IncrementalNode(module: module, prefix: prefix)
 
+            var testGenFile: String? = nil
+            if module is TestModule {
+                testGenFile = Path.join(xcTestGenPath, "\(module.c99name)-XCTestManifest")
+            }
+                // let testGenFile = 
+                // sources += [Path.join(xcTestGenPath, "\(module.c99name)-XCTestManifest.swift")]
+            // }
+
             try write("  ", module.targetName, ":")
             try write("    tool: swift-compiler")
             try write("    executable: ", Resources.path.swiftc)
             try write("    module-name: ", module.c99name)
             try write("    module-output-path: ", node.moduleOutputPath)
             try write("    inputs: ", node.inputs)
-            try write("    outputs: ", node.outputs)
+            try write("    outputs: ", node.outputs + (testGenFile != nil ? ["\(testGenFile!).o"] : []))
             try write("    import-paths: ", prefix)
             try write("    temps-path: ", node.tempsPath)
-            try write("    objects: ", node.objectPaths)
+            try write("    objects: ", node.objectPaths + (testGenFile != nil ? ["\(testGenFile!).o"] : []))
             try write("    other-args: ", args + otherArgs)
-            try write("    sources: ", module.sources.paths)
+            try write("    sources: ", module.sources.paths + (testGenFile != nil ? ["\(testGenFile!).swift"] : []))
```

Comment:
Are you assuming that tests will never be compiled in Release mode? Otherwise you'd need to add the new generated files down in the .Release branch as well.